docker-compose.yml

```diff
@@ -33,12 +33,14 @@ services:
 #      PHP_XDEBUG: 1
 #      PHP_XDEBUG_MODE: debug
 #      PHP_XDEBUG_MODE: profile
+#      PHP_XDEBUG_START_WITH_REQUEST: "yes"
 #      PHP_XDEBUG_USE_COMPRESSION: false
 #      PHP_IDE_CONFIG: serverName=my-ide
 #      PHP_XDEBUG_IDEKEY: "my-ide"
 #      PHP_XDEBUG_CLIENT_HOST: 172.17.0.1 # Linux
 #      PHP_XDEBUG_CLIENT_HOST: host.docker.internal # Docker 18.03+ Mac/Win
 #      PHP_XDEBUG_CLIENT_HOST: 10.0.75.1 # Windows, Docker < 18.03
+#      PHP_XDEBUG_LOG_LEVEL: 0
```

Comment:
As an option, we can provide both options similarly to `PHP_XDEBUG_MODE` environment variable.
```suggestion
#      PHP_XDEBUG_LOG_LEVEL: 0
#      PHP_XDEBUG_LOG_LEVEL: 3
```

Reply:
```suggestion
```